ts/src/pro/binance.ts

```diff
@@ -1216,10 +1220,14 @@ export default class binance extends binanceRest {
          * @method
          * @name binance#watchOHLCVForSymbols
          * @description watches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+         * @see https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data
+         * @see https://binance-docs.github.io/apidocs/futures/en/#kline-candlestick-data
+         * @see https://binance-docs.github.io/apidocs/delivery/en/#kline-candlestick-data
          * @param {string[][]} symbolsAndTimeframes array of arrays containing unified symbols and timeframes to fetch OHLCV data for, example [['BTC/USDT', '1m'], ['LTC/USDT', '5m']]
          * @param {int} [since] timestamp in ms of the earliest candle to fetch
          * @param {int} [limit] the maximum amount of candles to fetch
          * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {object} [params.timezone] if provided, kline intervals are interpreted in that timezone instead of UTC, example '+08:00'
```

Comment:
@carlosmiei imo here should be`8` suggested instead of `'+08:00'`, because in long run we should make users get used to unified approach, while +08:00 is a no go imo, neither unified value

Reply:
for now that's not unified so we can keep it I think, in the future if we unify timezones we can choose a more unified syntax